README.md

```diff
@@ -0,0 +1,75 @@
+[![Build Status](https://travis-ci.org/kuzzleio/sdk-java.svg?branch=master)](https://travis-ci.org/kuzzleio/sdk-java)
+
+Official Kuzzle JAVA SDK
+======
+
+## About Kuzzle
+
+A backend software, self-hostable and ready to use to power modern apps.
+
+You can access the Kuzzle repository on [Github](https://github.com/kuzzleio/kuzzle)
+
+## SDK Documentation
+
+The complete SDK documentation is available [here](http://docs.kuzzle.io/sdk-reference/)
+
+## Protocol used
+
+The JAVA SDK implement the websocket protocol.
+
+## Build
+
+### Clone the depot
```

Comment:
depot => repository

Better yet, I think that this subtitle can be removed. It doesn't seem to be useful in that section, given that this is the only subtitle, and because the snippet below does not only clone the repo, it also builds it.

My proposal: replace it with a simple comment: `Execute the following snippet to clone the GIT repository, and build the SDK. It will then be available in the "build/java/build/libs" directory`

(if you agree with my proposal, then the sentence following the snippet should be removed)